APE_specutils.rst

```diff
@@ -0,0 +1,147 @@
+
+The next step: specreduce and specviz
+++++++++++++++++++++
+This APE is not intended to specify the full details of the specvis and specreduce packages, but we provide a broad-brush description of what the two packages include.  They will each have their own separate APEs in the future.
+
+Common data reduction requirements for optical spectra:
+
+* Methods for extracting a 1D spectrum
+* Wavelength calibration
+* Removal of sky features
+* Flux calibration of spectra
+
+Common visualization requirements:
+
+* Be able to display a spectra
+* Being able to explore the spectra by zooming in on features or moving to different areas. 
+* Being able to examine the details of the spectra
+* Integrate tools into visualization to interactively perform analysis
+
+
+
+Branches and pull requests
+--------------------------
+N/A
+
+
+Implementation
+--------------
+
+`specutils <https://github.com/astropy/specutils>`_ already exists in its github repository, although the changes outlined in this APE will need to be implemented there. `specreduce <https://github.com/crawfordsm/specreduce>`_ also has an implementation, which can be adapted to use the framework described here.  Similary, visualization via the `specviz <https://github.com/spacetelescope/specreduce>`_ package will do the same.  Of course, other packages for spectroscopic analysis exist, and hopefully will also be adapted to this framework in due time.
+
+
+Backward compatibility
+----------------------
+
+The proposed changes will likely break backwards compatibility of the `specutils` package.  To alleviate this there will be a release of `specutils` prior to changes proposed in this APE.
+
+
+Alternatives
+------------
+
+Despite the existence of an astropy `specutils` package with a `Spectrum1D` object, it has not been widely adopted.  This is the original motivation for this APE.  However, there are other options to consider:
+
+1. Do nothing.  Pro:  This requires no additional work or collaboration.   Con:   We will continue to redevelop the same set of existing tools and waste limited resources in the community.  There will also be in clear, established way of teaching how to develop spectral objects in python. 
+2. Have the Spectrum1D object be a very limited base class and have development for different  discipline/wavelenth ranges happen independently (i.e.  RadioSpectrum1D, OpticalSpectrum1D, XRaySpectrum1D would have their own packages).   Pro:  This is more descriptive and then more familiar for users of a field. Con:  This will likely lead to redevelopment of some tools, and likely lead to less sharing of  tools developed for different domains. 
+3. Developing functional code with no Spectrum1D object. Pro:  Faster development with less overhead.  Con:  Increase difficulty in maintaining common namespace (wave vs. wavelength vs. something else) that can lead to conflicts in collaborating. Also is quite different from other Astropy efforts and Python standards (e.g. PEP8).
```

Comment:
I guess it refers to the astropy coding style guide (that is very broad): http://docs.astropy.org/en/latest/development/codeguide.html#coding-style-conventions